lib/rspec/rails/matchers/active_job.rb

```diff
@@ -149,12 +149,17 @@ def other_attributes_match?(job)
 
           def serialized_attributes
             {}.tap do |attributes|
-              attributes[:at]    = @at.to_f if @at
+              attributes[:at]    = serialized_at
```

Comment:
This is not quite the same, can we do  `serialized_at if @at` and eliminate the nil check in the method, we don't want to se this at all if it hasn't been set.